tests/Processor/SanitizeCookiesProcessorTest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Raven\Tests\Processor;
+
+use PHPUnit\Framework\TestCase;
+use Raven\Client;
+use Raven\ClientBuilder;
+use Raven\Event;
+use Raven\Processor\SanitizeCookiesProcessor;
+
+class SanitizeCookiesProcessorTest extends TestCase
+{
+    /**
+     * @var Client
+     */
+    protected $client;
+
+    protected function setUp()
+    {
+        $this->client = ClientBuilder::create()->getClient();
+    }
+
+    /**
+     * @expectedException \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     * @expectedExceptionMessage You can configure only one of "only" and "except" options.
+     */
+    public function testConstructorThrowsIfBothOnlyAndExceptOptionsAreSet()
+    {
+        new SanitizeCookiesProcessor([
+            'only' => ['foo'],
+            'except' => ['bar'],
+        ]);
+    }
+
+    /**
+     * @dataProvider processDataProvider
+     */
+    public function testProcess($options, $inputData, $expectedData)
+    {
+        $event = new Event($this->client->getConfig());
+        $event = $event->withRequest($inputData);
+
+        $processor = new SanitizeCookiesProcessor($options);
+        $event = $processor->process($event);
+
+        $this->assertArraySubset($expectedData, $event->getRequest());
+    }
+
+    public function processDataProvider()
```

Comment:
Since the only differences between the data is in cookies and options, maybe you can remove all the other stuff and move the generic request data into the test.